src/components/project/sections/gallery-2.tsx

```diff
@@ -1,25 +1,61 @@
 import Image from "next/image";
 import { Gallery } from "../../../interfaces/project";
+import styled from "@emotion/styled";
+import { breakpoints } from "../../../styles/variables";
+import useIsDesktop from "../../../hooks/useIsDesktop";
+import useIsTablet from "../../../hooks/useIsTablet";
 
 interface IProps {
   images: Gallery;
 }
+const StyledImageWrapper = styled.div`
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 180px;
+  & img {
+    object-fit: cover;
+    width: auto;
+    border-radius: 26px;
+    &:not(:last-child) {
+      margin-right: 20px;
+    }
+  }
+
+  @media (max-width: ${breakpoints.md}px) {
+    flex-direction: column;
+    margin-bottom: 120px;
+    & img {
+      // object-fit: cover;
+      height: auto;
+      width: 100%;
+      border-radius: 26px;
+      &:not(:last-child) {
+        margin-bottom: 8px;
+        margin-right: 0px;
+      }
+    }
+  }
+`;
 
 export default function Gallery2({ images }: IProps) {
+  const isDesktop = useIsDesktop();
+  const isTablet = useIsTablet();
   return (
-    <div>
+    <StyledImageWrapper>
       {images.nodes.map((image, index) => {
         return (
           <Image
             key={index}
-            width={500}
-            height={200}
+            width={isDesktop ? 500 : isTablet ? 686 : 343}
+            height={isDesktop ? 768 : isTablet ? 428 : 236}
```

Comment:
I'm getting an warning in the console with these images, could you investigate a bit:

`Image with src "http://web-space.local/wp-content/uploads/2024/02/Rectangle-8.png" has a "loader" property that does not implement width. Please implement it or use the "unoptimized" property instead.`